docs/native/src/overview.rst

```diff
@@ -1,5 +1,76 @@
 
 ..  _intro:
 
-|stub-icon| Integrating with Office for iOS
-===========================================
+Integrating with Office for iOS
+===============================
+You can integrate with Office for iOS to enable your users to view and edit Excel, PowerPoint, and Word files directly on the iPad and iPhone.
+
+.. image:: /images/ScenarioEdit.png
+   :alt: A screenshot that shows editing a file in Office for iOS.
+   :align: center
+
+If you deliver a iOS-based experience that allows your users to store Office files or includes Office files as a key part of your solution, you now have the opportunity to integrate Office for iOS into your experience. This integration works directly against files stored by you. Your users won't need a separate storage solution to view and edit Office files.
+
+Editing & Viewing Office files
+------------------------------
+When you integrate with Office for iOS, your users can view and edit Excel, PowerPoint, and Word files directly on the iOS device. The level of editing support is determined by the type of user and whether they have an O365 subscription. Here is a table of the key points about edit support. Please note this only applies to devices smaller than 10.1".
+
+..  note::
+    * MSA Free: the user has a Microsoft account, but no O365 subscription
+    * MSA Subscription: the user has a Microsoft account and an individual O365 subscription
+    * Org subscription: the user belongs to an organization which has an O365 subscription
+    * Premium editing: refers to certain Office for iOS edit functionality which requires an O365 subscription to use
+
+================= ===============  ===============   ===============
+ACCOUNT TYPE      Consumer         Enterprise        EDU
+================= ===============  ===============   ===============
+Not signed in     View only	       View only	       Free edit
+MSA Free          Free Edit	       View only	       Free edit
+MSA Subscription  Premium editing  Premium editing	 Premium editing
+Org Subscription  Premium editing  Premium editing	 Premium editing
+================= ===============  ===============   =============== 
+
+
+Integration process
```

Comment:
Given that most of this is repeating existing content, I recommend refactoring it into a shared snippet so that it's easier to maintain.